v4/source/channels.yaml

```diff
@@ -304,6 +375,26 @@
           $ref: '#/responses/Forbidden'
         '404':
           $ref: '#/responses/NotFound'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            patch := &model.ChannelPatch{
+              Name:        new(string),
+              DisplayName: new(string),
+              Header:      new(string),
+              Purpose:     new(string),
+            }
+            *patch.Name = model.NewId()
+            *patch.DisplayName = model.NewId()
+            *patch.Header = model.NewId()
+            *patch.Purpose = model.NewId()
```

Comment:
Maybe instead of `model.NewId()` we could do `"somename"`, `"somedisplayname"`, etc. I think if someone doesn't know what `model.NewId()` is that could cause confusion